doc/Tex/RAM_SCB_scripts.tex

```diff
@@ -1,73 +1,39 @@
 
-All scripts are designed to be called from the command line and have a common
-help feature:
+All scripts are designed to be called from the command line and have a common help feature:
 \begin{verbatim}
 ScriptName -h
 \end{verbatim}
-\noindent will print out the script's help text.  This info is typically 
-more complete and up-to-date than the info listed here.
+\noindent will print out the script's help text.  This info is typically more complete and up-to-date than the info listed here.
 
-While all RAM-SCB scripts are listed here, only a handful of SWMF scripts are
-described.  Be sure to explore the {\tt Share/Scripts} directory before you
-constructing your own solutions to common problems (e.g. endian conversions, 
-PARAM-checking, etc.) 
+While all RAM-SCB scripts are listed here, only a handful of SWMF scripts are described. Be sure to explore the {\tt Share/Scripts} directory before you constructing your own solutions to common problems (e.g. endian conversions, PARAM-checking, etc.) 
 
 \section{Config.pl (SWMF Script)}
-{\tt Config.pl} is part of the SWMF \textit{Config} system for installing and
-pre-configuring RAM-SCB in both stand-alone and component modes.  
-Use of {\tt Config.pl} is covered extensively in Chapter \ref{subchap:install}.
+{\tt Config.pl} is part of the SWMF \textit{Config} system for installing and pre-configuring RAM-SCB in both stand-alone and component modes. Use of {\tt Config.pl} is covered extensively in Chapter \ref{subchap:install}.
 
 \section{DiffNum.pl (SWMF Script)}
-{\tt DiffNum.pl} is a powerful, quantitative re-write of the popular {\tt diff}
-utility.  It compares two files, finds and quantifies differences in any
-numerical entries, and, if any are found, lists the differences and raises an
-exception.  The main purpose of {\tt DiffNum.pl} is to find and quantify 
-failures in RAM-SCB tests.
+{\tt DiffNum.pl} is a powerful, quantitative re-write of the popular {\tt diff} utility.  It compares two files, finds and quantifies differences in any numerical entries, and, if any are found, lists the differences and raises an exception.  The main purpose of {\tt DiffNum.pl} is to find and quantify failures in RAM-SCB tests.
 
 Usage:
 \begin{verbatim}
 DiffNum.pl [options] File1 File2
 \end{verbatim}
 
-Common options include {\tt -a=VALUE} and {\tt -r=VALUE}, which allow the user
-to ignore absolute and relative differences less than {\tt VALUE} and {\tt -t}
-which turns off the comparison of text.
+Common options include {\tt -a=VALUE} and {\tt -r=VALUE}, which allow the user to ignore absolute and relative differences less than {\tt VALUE} and {\tt -t} which turns off the comparison of text.
 
 
 %\section{GetOmni.py (RAM-SCB Script) \label{subchap:getomni}}
 %Script forthcoming.
+% QUESTION: We sort of have this now, should we make a section for it?

```

Comment:
Yes we should add a section, but we don't have this *in* in the repo, do we? We should make sure the script is in the repo, then make sure the section documenting use is present and correct.

Reply:
I don't believe the script is currently in the repo, good point that it probably should be if we are going to document it